src/components/layoutElementWithTitle/style.ts

```diff
@@ -4,6 +4,7 @@ import { ILayoutElementWithTitleProps } from "components/layoutElementWithTitle/
 export const SWrapper = styled("div")`
   display: flex;
   align-items: flex-start;
+  flex-direction: column;
```

Comment:
Check please create job post page and freelancer edit profile page, and two another pages which uses this component. I believe that this one line will change user interface on all forms which maded before!!! Use media query with max width to implement this style only for cases when it small width, which is will work in this case

Reply:
Ohh, I forgot to change it